src/Servers/HttpSys/src/HttpSysOptions.cs

```diff
@@ -38,7 +38,7 @@ public string RequestQueueName
             get => _requestQueueName;
             set
             {
-                if (value.Length > MaximumRequestQueueNameLength)
+                if (value?.Length > MaximumRequestQueueNameLength)
```

Comment:
Was this just a random find?

Reply:
RequestQueueName was added as part of this feature, just in an earlier PR. And yeah, when writing tests I accidentally tried setting it to null and it threw null ref.